lib/applications/resolvers.ts

```diff
@@ -713,3 +717,131 @@ export const generateApplicationsReport: Resolver = async (_, args, { prisma })
+      addressLine2: columnsSet.has(PermitHoldersReportColumn.HomeAddress),
+      city: columnsSet.has(PermitHoldersReportColumn.HomeAddress),
+      province: columnsSet.has(PermitHoldersReportColumn.HomeAddress),
+      postalCode: columnsSet.has(PermitHoldersReportColumn.HomeAddress),
+      email: columnsSet.has(PermitHoldersReportColumn.Email),
+      phone: columnsSet.has(PermitHoldersReportColumn.PhoneNumber),
+      guardianFirstName: columnsSet.has(PermitHoldersReportColumn.GuardianPoaName),
+      guardianMiddleName: columnsSet.has(PermitHoldersReportColumn.GuardianPoaName),
+      guardianLastName: columnsSet.has(PermitHoldersReportColumn.GuardianPoaName),
+      guardianRelationship: columnsSet.has(PermitHoldersReportColumn.GuardianPoaRelation),
+      guardianAddressLine1: columnsSet.has(PermitHoldersReportColumn.GuardianPoaAddress),
+      guardianAddressLine2: columnsSet.has(PermitHoldersReportColumn.GuardianPoaAddress),
+      guardianPostalCode: columnsSet.has(PermitHoldersReportColumn.GuardianPoaAddress),
+      guardianCity: columnsSet.has(PermitHoldersReportColumn.GuardianPoaAddress),
+      guardianProvince: columnsSet.has(PermitHoldersReportColumn.GuardianPoaAddress),
+      permit: {
+        select: {
+          rcdPermitId: true,
+          active: columnsSet.has(PermitHoldersReportColumn.UserStatus),
+        },
+      },
+      permitType: columnsSet.has(PermitHoldersReportColumn.RecentAppType),
+    },
+  });
+
+  // Formats the date fields and adds totalAmount, applicantName and rcdPermitId properties to allow for csv writing
+  const csvApplications = applications.map(application => {
+    return {
+      ...application,
+      dateOfBirth: formatDate(application.dateOfBirth),
+      applicantName:
+        application.firstName +
+        (application.middleName
+          ? ` ${application.middleName} ${application.lastName}`
+          : ` ${application.lastName}`),
+      rcdPermitId: application.permit?.rcdPermitId,
+      homeAddress: `${application.addressLine1}, 
+      ${application.addressLine2 ? `${application.addressLine2},` : ''} ${application.city}, ${
+        application.province
+      }, ${application.postalCode}`,
+      userStatus: application.permit?.active ? 'Active' : 'Inactive',
+      guardianPOAName:
+        application.guardianFirstName +
+        (application.guardianMiddleName
+          ? ` ${application.guardianMiddleName} ${application.guardianLastName}`
+          : ` ${application.guardianLastName}`),
+      guardianPOAAdress: `${
+        application.guardianAddressLine1 ? `${application.guardianAddressLine1},` : ''
+      } ${application.guardianAddressLine2 ? `${application.guardianAddressLine2},` : ''} ${
+        application.guardianCity ? `${application.guardianCity},` : ''
+      } ${application.guardianProvince ? `${application.guardianProvince},` : ''} ${
+        application.guardianPostalCode ?? ''
+      }`,
```

Comment:
This is the value that's written to the csv for guardian/poa address.
This separates the address values with commas (and all the address values are nullable so I used a ternary operator). This doesn't look great but works lol, lmk if you have any suggestions to improve this.

Reply:
This works haha, perhaps we can make it a bit readable using my above comment, but no big deal if we can't

Reply:
Addressed with the following code
![image](https://user-images.githubusercontent.com/51224641/147161046-60032fd8-4101-462a-8d39-21b62534324b.png)